Lib/socket.py

```diff
@@ -591,8 +591,9 @@ def readinto(self, b):
                 self._timeout_occurred = True
                 raise
             except error as e:
-                if e.args[0] in _blocking_errnos:
-                    return None
+                if e.args:
```

Comment:
You can write the check in one line, ``if e.args and e.args[0] in _blocking_errnos:``

Reply:
you right.
I changed. Thank you